3.10/quick-start-in-the-cloud.md

```diff
@@ -140,12 +140,14 @@ Alternatively, follow the steps of the linked guides:
 
 ## Free-to-Try vs. Professional Service
```

Comment:
This heading needs to change. Would suggest "Free-To-Try vs. Paid".

Reply:
Done.